<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShutterTG</class>
 <widget class="QDialog" name="ShutterTG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gbShutterTG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>516</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>205</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Shutter for repeats</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDownload">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>420</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Download sequence to MIPS</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>135</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TsD, Shutter D</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tp, period, mS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTdB">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>95</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>675</x>
      <y>55</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tdf, mS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTsB">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>95</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTdD">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>135</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>420</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;MIPS digital I/O map&lt;br/&gt;A = Shutter A&lt;br/&gt;B = Shutter B&lt;br/&gt;C = Shutter C&lt;br/&gt;D = Shutter D&lt;br/&gt;E = Sync out&lt;br/&gt;F = A, ADC trigger&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>695</x>
      <y>255</y>
      <width>106</width>
      <height>76</height>
     </rect>
    </property>
    <property name="text">
     <string>Tp must exceed the largest Tdn value.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTsD">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>135</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSelectedShutter">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>200</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxCount">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Sutter pulse width to 0 to disable</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>95</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TsB, Shutter B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>55</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ts, uS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>385</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Full series repeat count</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>450</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Press start to start generation of pulse sequence or apply trigger to R input</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>115</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TsC, Shutter C</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbSequenceRepeats">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>236</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Sequence repeats</string>
    </property>
    <widget class="QComboBox" name="comboSelectN">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="maxCount">
      <number>2000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select n</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tdn, in mS</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTdn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>m, repeat count</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leM">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbPaste">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>105</y>
       <width>206</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Paste sequence data from the clipboard, must be text, you can not paste from excel. Copy from excel to a text file and then copy from the text file and paste here.</string>
     </property>
     <property name="text">
      <string>Paste text from clipboard</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>75</y>
      <width>96</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TsA, Shutter A</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTdC">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>115</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>175</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>n, number of sequences</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTsC">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>115</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFullRepeat">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>385</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTp">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>25</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>226</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum sutter pulse width is 20uS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leN">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>170</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Can not exceed 2000</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTsA">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>75</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbShow">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>420</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Pulse Sequence</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="files.qrc">:/Images/ShutterTG.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDefScan">
    <property name="geometry">
     <rect>
      <x>695</x>
      <y>335</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Def Scan</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
